e2e_test/js-tests/test_viem_tx.mjs

```diff
@@ -63,7 +79,8 @@ const testNonceBump = async (
 			account,
 			to: "0x00000000000000000000000000000000DeaDBeef",
 			value: 3,
-			gas: 90000,
+			// dynaimcally retrieve intrinsic gas from feeCurrency directory "feeCurrencyConfig"
```

Comment:
Do you still plan to implement this for this PR? If not, I would remove the line or add a `TODO:`, because it doesn't reflect what's happening ;)

Reply:
Ok removed and I added a ticket for it - https://github.com/celo-org/op-geth/issues/260